runtime/compiler/optimizer/TreeLowering.cpp

```diff
@@ -1,4 +1,5 @@
 /*******************************************************************************
+
```

Comment:
Is this extra empty line intended?

Reply:
Thanks for catching that!

Reply:
Fixed in commit 3ddc5afce0d07fe7ad70e44b6a840d490f85aa49